app/src/ereferrals/java/org/eyeseetea/malariacare/data/database/utils/PreferencesEReferral.java

```diff
@@ -60,6 +60,22 @@ public static long getUserProgramId() {
         return sharedPreferences.getLong(context.getString(R.string.logged_user_program), -1);
     }
 
+    public static void saveLastDownloadedProgramMedia(String programName) {
```

Comment:
I think that program associate to media should be program associated to the user then these methods are not necessary